frontend/src/component/project/Project/ProjectFeatureToggles/ActionsCell/ActionsCell.tsx

```diff
@@ -9,20 +9,30 @@ import {
     Popover,
     Tooltip,
     Typography,
+    styled,
 } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
 import MoreVertIcon from '@mui/icons-material/MoreVert';
 import FileCopyIcon from '@mui/icons-material/FileCopy';
 import ArchiveIcon from '@mui/icons-material/Archive';
 import WatchLaterIcon from '@mui/icons-material/WatchLater';
-import { useStyles } from './ActionsCell.styles';
 import { PermissionHOC } from 'component/common/PermissionHOC/PermissionHOC';
 import {
     CREATE_FEATURE,
     DELETE_FEATURE,
     UPDATE_FEATURE,
 } from 'component/providers/AccessProvider/permissions';
 
+const StyledBoxCell = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+    paddingRight: theme.spacing(2),
+}));
+
+const StyledMenuItem = styled(MenuItem)(({ theme }) => ({
```

Comment:
would styled<MenuItem>(... work as well?
as is really a type casting that may lead to type errors. I'd use it as a last resort